manifests/h/HydrusNetwork/HydrusNetwork/v608/HydrusNetwork.HydrusNetwork.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: HydrusNetwork.HydrusNetwork
+PackageVersion: v608
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: Hydrus Network_is1
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```

I'm curious if this was automatically modified by wingetcreate? 🤔 If yes, it may be worth opening an issue over at [winget-create's repo](https://github.com/microsoft/winget-create)

Regardless if it is a wingetcreate issue, next time you can provide the architecture manually in the URL with arch overrides as:

`wingetcreate update HydrusNetwork.HydrusNetwork --version <version> --urls '<InstallerUrl>|x64'`

---

yes, could be my mistake, i think previous submitter was using `|x64`